paasta_tools/marathon_tools.py

```diff
@@ -1112,3 +1114,27 @@ def get_num_at_risk_tasks(app, draining_hosts):
             num_at_risk_tasks += 1
     log.debug("%s has %d tasks running on at-risk hosts." % (app.id, num_at_risk_tasks))
     return num_at_risk_tasks
+
+
+def broadcast_log_all_services_running_here(line, component='monitoring'):
```

Comment:
This is a good start. Eventually I would like this to be more than just marathon. For example that other srv-configs io ticket would be easier to troubleshoot if we just saw in the logs that there was a shutdown right before. But that is a chronos job.

Could you use `mesos_services_running_here` instead?

Reply:
@solarkennedy take a look. We filter frameworks to allow both marathon* and chronos* - is this what you meant?

Reply:
Sure, but how about not filtering at all?